### PR TITLE
370097: Migrate Walleys Quarry to ADP

### PR DESCRIPTION
**Pull request by santosh-gh**
AB#370097

# **What this PR does / why we need it**:
Migrating the existing Walleys Quarry project to ADP platform

# **Special notes for your reviewer**
Files and code are added and removed from the existing  project to compatible with ADP platform
Set CI pipeline for build and publish app and helm chart

# Testing
Any relevant testing information and pipeline runs.
https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=563823&view=results
https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=563170&view=results
ea-wq-incident-web.snd3.adp.defra.gov.uk

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [x] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:

### CODE REVIEW

File: .azuredevops/build.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - repository: DEFRA-ADPPipelineCommon
+      name: DEFRA/adp-pipeline-common
+      endpoint: DEFRA
+      type: github
+      ref: main
+
+extends:
+  template: /pipelines/common-app-build.yaml@DEFRA-ADPPipelineCommon
+  parameters:
+    projectName: "ea-wq-incident-web"
```
Comment (rajeshkaremane) [inline .azuredevops/build.yaml:44]: Project name is not in use please add programmeName="ea"

File: appConfig/appConfig.snd3.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- key: APPINSIGHTS_INSTRUMENTATIONKEY
```
Comment (rajeshkaremane) [inline appConfig/appConfig.snd3.yaml:1]: Check with Asad if snd4 is availble?

File: .azuredevops/build.yaml (hunk 1 above)
Comment (santosh-gh) [reply .azuredevops/build.yaml]: As discussed with Ken, the programName is optional so not included

File: appConfig/appConfig.snd3.yaml (hunk 2 above)
Comment (santosh-gh) [reply appConfig/appConfig.snd3.yaml:1]: Checked with Asad, it is not 100% ready, so not included now

**Review by kbabigumira — COMMENTED** [review]
Missing the catalog file

File: helm/ea-wq-incident-web/values.yaml [hunk 3]
```diff
@@ -1,26 +1,10 @@
 environment: development
 name: ea-wq-incident-web
-namespace: ea-wq
 image: ea-wq-incident-web
-workstream: ea-wq
 
 labels:
```
Comment (kbabigumira) [inline helm/ea-wq-incident-web/values.yaml:5]: No health points?

Comment (santosh-gh) [reply helm/ea-wq-incident-web/values.yaml:7]: Added catalog file

Comment (santosh-gh) [reply helm/ea-wq-incident-web/values.yaml:7]: updated health probes

**Review by kbabigumira — APPROVED** [review]

**Review by DaRoszko — APPROVED** [review]

**Pull request merged**